src/contract_wrappers/modules/transfer_manager/__tests__/general_transfer_manager_wrapper.test.ts

```diff
@@ -525,6 +540,856 @@ describe('GeneralTransferManagerWrapper', () => {
     });
   });
 
+  describe('GetInvestorFlag', () => {
+    test('should getInvestorFlag', async () => {
+      const mockedParams = {
+        investor: '0x3333333333333333333333333333333333333333',
+        flag: FlagsType.IsAccredited,
+      };
+      // Address expected
+      const expectedResult = true;
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.getInvestorFlag).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync(mockedParams.investor, mockedParams.flag)).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.getInvestorFlag(mockedParams);
+
+      // Result expectation
+      expect(result).toBe(expectedResult);
+
+      // Verifications
+      verify(mockedContract.getInvestorFlag).once();
+      verify(mockedMethod.callAsync(mockedParams.investor, mockedParams.flag)).once();
+    });
+  });
+
+  describe('GetAllInvestorFlags', () => {
+    test('should getAllInvestorFlags', async () => {
+      // Address expected
+      const expectedResult = [['0x3333333333333333333333333333333333333333'], [new BigNumber(2)]];
```

Comment:
Would maybe be good to have more then one investor in test array

Reply:
sounds good